### PR TITLE
updated to Webpack2

### PR DESCRIPTION
**Pull request by kuldeepkeshwar**

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -1,10 +1,67 @@
-import webpack from 'webpack';
-import ExtractTextPlugin from 'extract-text-webpack-plugin';
-import HtmlWebpackPlugin from 'html-webpack-plugin';
-import autoprefixer from 'autoprefixer';
+const webpack = require('webpack');
```
Comment (developit) [inline webpack.config.babel.js]: Did these have to change because of the `babelrc`?

Comment (kuldeepkeshwar) [inline webpack.config.babel.js]: yes I was using `"presets": [["es2015", { "modules": false }]]` then it was not able to understand `import` statement (`SyntaxError: Unexpected token import`).
somehow I forgot to commit `babelrc` :-)

Comment (developit) [inline webpack.config.babel.js]: heh - figured as much.

**Comment by developit** [comment]
I ported this over to preact-cli for the performance and simplicity benefits, so it's now technically built with Webpack 3.

**Pull request closed without merging**